android/src/main/cpp/NativeProxy.h

```diff
@@ -66,8 +66,9 @@ class EventHandler : public HybridClass<EventHandler> {
 
   void receiveEvent(
       jni::alias_ref<JString> eventKey,
+      jint targetTag,
```

Comment:
Since we already have `emitterReactTag`, does it make sense to rename this to `targetReactTag` as well?
```suggestion
      jint targetReactTag,
```

Reply:
Thanks, i missed it! I think that more relevant will be `emitterReactTag` instead of `targetReactTag` here, what do you think?